docs: add docs for filesPath built-in function (#153)
reviewpad/docs: docs/guides/built-ins/functions/filesPath/index.mdx

import CodeBlock from "@theme/CodeBlock";
import Example from "!!raw-loader!./example.yml";

**Description**:

> Retrieves a list of file paths changed by the pull request.

Reviewer: ```suggestion
Retrieves the list of all file paths changed in the pull request.
```

:::caution

This buit-in is **not** available for [`issue`](/guides/syntax#workflow).

:::

**Parameters**:

_none_

**Return value**:

| Type       | Description                                             |
| :--------- | :------------------------------------------------------ |

> | `[]string` | A list of file paths of the changed files in the patch. |

Reviewer: ```suggestion
| `[]string` | A list of all file paths changed files in the patch. |
```
